src/ir/ty.rs

```diff
@@ -464,14 +464,14 @@ impl DotAttributes for TypeKind {
 
 #[test]
 fn is_invalid_named_type_valid() {
```

Comment:
These tests should replace `named_type` with `type_param` in their function names.